inst/include/distributions/functors/lognormal_lpdf.hpp

```diff
@@ -13,22 +25,30 @@ namespace fims_distributions
     template <typename Type>
     struct LogNormalLPDF : public DensityComponentBase<Type>
     {
-        fims::Vector<Type> log_sd;
-        fims::Vector<Type> mu;
-        fims::Vector<Type> sd;
-        std::vector<bool> is_na;
+        fims::Vector<Type> log_sd; /*!< log of the standard deviation of the distribution on the log scale; can be a vector or scalar */
+        fims::Vector<Type> mu; /*!< mean of the distribution on the log scale; can be a vector or scalar */
+        fims::Vector<Type> sd; /*!< standard deviation of the distribution on the log scale; can be a vector or scalar */
+        std::vector<bool> is_na; /*!< Boolean; if true, data observation is NA and the likelihood contribution is skipped */
         #ifdef TMB_MODEL
-        ::objective_function<Type> *of;
+        ::objective_function<Type> *of; /*!< Pointer to the TMB objective function */
         #endif
-        Type nll = 0.0;
-        // data_indicator<tmbutils::vector<Type> , Type> keep;
+        Type nll = 0.0; /*!< total negative log-likelihood contribution of the distribution */
+        // data_indicator<tmbutils::vector<Type> , Type> keep; /*!< Indicator used in TMB one-step-ahead residual calculations */
 
+        /** @brief Constructor.
+         */
         LogNormalLPDF() : DensityComponentBase<Type>()
         {
         }
 
+        /** @brief Destructor.
+         */
         virtual ~LogNormalLPDF() {}
 
+        /**
+         * @brief Evaluates the negative log-likelihood of the lognormal probability density function
+         * @param do_log Boolean; if true, log densities are returned
```

Comment:
If I am being overly pedantic, please ignore this comment. Are log densities actually returned? Or are log densities used in the calculation of the negative log likelihood, which is returned? Apologies if I am mis-interpreting the intent of this documentation. This same comment applies to the other functions as well, but is not repeated in those other places.

Reply:
I have a note to refactor the do_log argument in sprint 2 so it is fixed at TRUE given how we apply these functions in FIMS